common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -2761,8 +2760,7 @@ object CelebornConf extends Logging {
       .doc("Max ratio of direct memory to store shuffle data")
       .version("0.2.0")
       .doubleConf
-      .checkValue(v => v >= 0.0 && v <= 0.4, "should be in [0.0, 0.4].")
-      .createWithDefault(0.1)
+      .createWithDefault(0.0)
```

Comment:
Is it OK to default to 0?

Reply:
For version 0.2.0 there is no memory shuffle storage yet. So the default value changed to 0 is safe now.

Reply:
But this PR is for main branch